package.json

```diff
@@ -222,7 +222,8 @@
         "command": "confluent.artifacts.flink-compute-pool.select",
         "icon": "$(confluent-flink-compute-pool)",
         "title": "Select Flink Compute Pool",
-        "category": "Confluent: Flink Artifacts"
+        "category": "Confluent: Flink Artifacts",
+        "enablement": "false"
```

Comment:
Without this, the command would still be visible from the command palette (but wouldn't do anything if clicked), so this is mainly a bit of cleanup.